modules/veteran_verification/veteran_verification.gemspec

```diff
@@ -0,0 +1,25 @@
+# frozen_string_literal: true
+
+$LOAD_PATH.push File.expand_path('../lib', __FILE__)
+
+# Maintain your gem's version:
+require 'veteran_verification/version'
+
+# Describe your gem and declare its dependencies:
+Gem::Specification.new do |s|
+  s.name        = 'veteran_verification'
+  s.version     = VeteranVerification::VERSION
+  s.authors     = ['Edward Paget']
+  s.email       = ['ed@adhocteam.us']
+  s.homepage    = 'https://api.vets.gov/services/veteran_verification/docs/v0/api'
+  s.summary     = 'Veteran Verification APIs'
+  s.description = 'Collection of API resources intended for 3rd verification of veteran status and service history'
+  s.license     = 'MIT'
```

Comment:
Not MIT license, see the creative commons public domain license in the root of vets-api. That is the correct license for a government work.